test/index.spec.ts

```diff
@@ -144,6 +145,43 @@ describe("addLambdaFunctions", () => {
       ],
     });
   });
+
+  it("adds aws-lambda-nodejs NodejsFunctions along with aws-lambda Functions", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "sa-east-1",
+      },
+    });
+    const hello1 = new lambda.Function(stack, "HelloHandler1", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const hello2 = new lambdaNodeJS.NodejsFunction(stack, "HelloNodeJsHandler", {
+      runtime: lambda.Runtime.NODEJS_14_X,
+      entry: "./test/lambda/example-lambda.ts",
+      handler: "handler",
+    });
+
+    let threwError = false;
+    try {
+      const datadogCdk = new Datadog(stack, "Datadog", {
+        nodeLayerVersion: 20,
+        pythonLayerVersion: 28,
+        addLayers: true,
+        enableDatadogTracing: false,
+        flushMetricsToLogs: true,
+        site: "datadoghq.com",
+        forwarderArn: "forwarder-arn",
+        apiKey: "1234",
+      });
+      datadogCdk.addLambdaFunctions([hello1, hello2]);
```

Comment:
Similar to my comment on the sample app, are `hello1` and `hello2` the most descriptive names here?